.gitignore

```diff
@@ -10,6 +11,11 @@ manual/source/*.table
 manual/source/classes/*/NX*.rst
 manual/source/classes/*/*.nxdl.xml
 manual/source/classes/contributed_definitions/canSAS
+manual/source/classes/applications/canSAS
```

Comment:
These items to be ignored in `manual/source/classes` can appear in local directories when the documentation is built in the source directory and not in the `build` directory.  We could use improvements in the `/Makefile` sections to remove these files before a `git add`, improvements that might help to avoid expanding this growing set of `.gitignore` items.

Reply:
Or we should not allow a build in the source directory.

Reply:
How would that be possible?

Reply:
By not adding that possibility to the Makefile? So `make local` becomes simply `make`.

Reply:
I'm actually surprised files are added to the source directory. That's not normal `sphinx` behaviour. I guess this is done by some custom code somewhere?

Reply:
`.rst` files are created (from documentation in NXDL files and the Schema) before sphinx is called.  Usually, the documentation is built out-of-source (outside the source tree).  An in-source build was always possible and that must be how these files were created locally.